src/middleware/express/make-middleware.ts

```diff
@@ -54,16 +58,22 @@ export function makeMiddleware<LoggerType>(
 
     const spanContext = getOrInjectContext(wrapper);
     const trace = `projects/${projectId}/traces/${spanContext.traceId}`;
+    const span = spanContext.spanId;
```

Comment:
can we extract isSampled field from the context? If the incoming request contains that, we should propagate its value downstream along with trace and span.

Reply:
yeah I think we should. will add it